aui.views/src/AUI/Platform/ATouchscreenKeyboardState.h

```diff
@@ -0,0 +1,35 @@
+// AUI Framework - Declarative UI toolkit for modern C++20
+// Copyright (C) 2020-2024 Alex2772 and Contributors
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+/**
+ * @brief State of touchscreen keyboard
+ */
+enum class ATouchscreenKeyboardState {
+    /**
+     * @brief Touchscreen keyboard state is unknown
```

Comment:
`UNKNOWN` is too common. Why didn't you used `NOOP` (no operation) as I've told you?

Reply:
NOOP is not a name for a touchscreen keyboard state